core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/orchestration/StartAmbariServicesRequest.java

```diff
@@ -8,15 +8,32 @@ public class StartAmbariServicesRequest extends StackEvent {
 
     private final boolean defaultClusterManagerAuth;
 
+    private boolean runPreServiceDeploymentForRecipe;
+
     @JsonCreator
     public StartAmbariServicesRequest(
             @JsonProperty("resourceId") Long stackId,
             @JsonProperty("defaultClusterManagerAuth") boolean defaultClusterManagerAuth) {
         super(stackId);
         this.defaultClusterManagerAuth = defaultClusterManagerAuth;
+        runPreServiceDeploymentForRecipe = true;
+    }
+
+    public StartAmbariServicesRequest(Long stackId, boolean defaultClusterManagerAuth,
+            boolean preServiceDeploymentForRecipe) {
+        this(stackId, defaultClusterManagerAuth);
+        runPreServiceDeploymentForRecipe = preServiceDeploymentForRecipe;
```

Comment:
It would be better to add the new field to the existing constructor and use that everywhere

Reply:
done